understanding/20/images-of-text.html

```diff
@@ -130,6 +134,14 @@ <h2>Examples of Images of Text</h2>
          <dt>Customizable font settings in images of text</dt>
          <dd>A Web site allows users to specify font settings and all images of text on the site
                are then provided based on those settings.</dd>
+         <dt>The text replicates an image of text</dt>
+         <dd>The CMS (content management system) allows content creators to incorporate both an image and a caption. 
+           While an image of text is utilized, the identical message is also presented as text directly beneath the image.</dd>
+           <dt>The text conveys the same meaning as the image of text</dt>
+         <dd>In a banner, textual information is combined with graphical content. Simply replicating 
```

Comment:
I'm having difficulty picturing this. We could mabye include an example? Otherwise, I'm inclined to not add this last example.